packages/components/src/text/next/use-text.js

```diff
@@ -55,7 +55,9 @@ export function useText( props ) {
 	if ( isHighlighter ) {
 		content = createHighlighterText( {
 			autoEscape: highlightEscape,
-			children,
+			// Disable reason: We need to disable this otherwise it erases the cast
+			// eslint-disable-next-line object-shorthand
+			children: /** @type {string} */ ( children ),
```

Comment:
If children must be a string, that should be part of the interface in the props type, right?

Reply:
It only _must_ be a string if `isHighlighter` is `true` 🤔